templates/repo/commit_author.tmpl

```diff
@@ -0,0 +1,56 @@
+{{/* Template Attributes:
+* User: The user associated with the commit, if any
+* Commit: The commit
```

Comment:
I don't think you should pass so many different structs.

`Commit` is not really used. What you need is a "git user" or "gitea user".